darwin/CpuFreq.h

```diff
@@ -0,0 +1,66 @@
+#ifndef HEADER_CpuFreq
+#define HEADER_CpuFreq
+
+#include "Machine.h"
+
+#if (defined(HAVE_LIBIOREPORT) && defined(HTOP_AARCH64))
```

Comment:
```suggestion
#if defined(HAVE_LIBIOREPORT) && defined(HTOP_AARCH64)
```
We're not writing Lisp …